hs_core/templates/pages/create-resource.html

```diff
@@ -3,153 +3,648 @@
+                                       data-dz-remove></i>
+                                </div>
+
+                                <span class="file-type-icon">
+
+                                </span>
+
+                                <div class="dz-error-message arrow_box arrow_box_top_center">
+                                    <span class="dz-error-message" data-dz-errormessage></span>
+                                </div>
+
+                                <div class="dz-filename" data-toggle="tooltip" data-placement="bottom"
+                                     title=""><span data-dz-name></span></div>
+                            </div>
+                        </div>
+
+                        <br>
+
+                        <div id="file-move-warning" class="alert alert-warning hidden-form">
+                            You selected move option which will move the selected files out of your iRODS account into a
+                            HydroShare resource once created.
+                        </div>
+
+                        {% if file_size_error %}
+                            <div class="alert alert-danger alert-dismissible">
+                                <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span
+                                        aria-hidden="true">&times;</span></button>
+                                {{ file_size_error }}
+                            </div>
+                        {% endif %}
+
+                        {% if validation_error %}
+                            <div id="file-type-error" class="alert alert-danger alert-dismissible">
+                                <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span
+                                        aria-hidden="true">&times;</span></button>
+                                {{ validation_error }}
+                            </div>
+                        {% endif %}
+                    </div>
+
+                    <div class="col-xs-12 col-sm-4 col-md-3">
+                        <ul class="list-group">
+                            <li class="list-group-item">Files you upload here will be grouped together into a
+                                "Resource" in HydroShare.
+                            </li>
+                            <li class="list-group-item">File size is limited to 1 GB per file for direct browser
+                                upload from your local disk.
+                            </li>
+                            <li class="list-group-item">Files larger than 1GB can be imported directly from any
+                                iRODS account. You can create
+                                a HydroShare iRODS account from your <a href="/user/{{ user.id }}/">User Profile</a>
+                                page.
+                                See <a href="https://pages.hydroshare.org/">Help</a> for information on working with

```

Comment:
@mauriel133 Change the help url to ```https://pages.hydroshare.org/creating-and-managing-resources/``` to better guide users for iRODS specific info.